src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/Browser/WasmTestWebServerStartup.cs

```diff
@@ -31,7 +31,8 @@ public void Configure(IApplicationBuilder app)
             app.UseStaticFiles(new StaticFileOptions
             {
                 FileProvider = new PhysicalFileProvider(s_hostingEnvironment.ContentRootPath),
-                ContentTypeProvider = provider
+                ContentTypeProvider = provider,
+                ServeUnknownFileTypes = true
```

Comment:
Maybe add the commit message here as a comment too, so, somebody in future doesn't end up removing that?

Reply:
heh true. I guess, I had ended up missing it the last time, but that was because of a comment! Ignore me 👍